seannkelleyy/Mario-Remake
# IH 113 add font and death message

﻿
# Description

New feat: Added death screens for a normal death and a game over.
Bug fix: Fixed issue where the game would not properly keep track of Mario's lives

closes #113 

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

# How To Test

Run the game and have Mario die normally and get a game over to see if the death screens properly display. 


# Checklist:
